src/routes/(split)/components/tooltip/+page.md

```diff
@@ -70,35 +70,35 @@ The tooltips will appear when the buttons are hovered or tapped.
         <button class="secondary" disabled>Disabled</button>
       </Tooltip>
       <Tooltip
-        id="example-button"
+        id="example-button-1"
         text={tooltipText}
       >
         <button class="secondary" disabled>Disabled</button>
       </Tooltip>
       <Tooltip
-        id="example-button"
+        id="example-button-2"
```

Comment:
FYI, another way to accomplish the same is to use
```
idPrefix="example-button"
```

Reply:
Good to know!